flocker/node/agents/functional/test_ebs.py

```diff
@@ -182,6 +188,132 @@ def test_next_device_in_use(self):
                                        {u"/dev/sdf"})
         self.assertEqual(result, u"/dev/sdg")
 
+    def test_create_volume_gold_profile(self):
```

Comment:
> Most of these tests seem to just duplicate the standard interface tests.

Slight variation: ``test_profile_respected`` tests that ``create_volume_with_profile`` with given profile succeeds in creating a volume. The EBS tests actually check that profile attributes reported by EBS backend match profile attributes published for ``MandatoryProfiles`` by EBS driver. Please let me know if that sounds reasonable? Thanks!